metadata/program/src/state/mod.rs

```diff
@@ -0,0 +1,9 @@
+const UNINITIALIZED_VERSION: u8 = 0;
+
+///prefix
```

Comment:
Please add a useful comment or better, make the const name more descriptive so a comment is unnecessary

Reply:
The analyzer throws an error if there isn't a comment there, so I add one. I assume this is a rust standard thing, being forced to have comments for every pub variable?

Reply:
You opted into this tyranny via https://github.com/solana-labs/solana-program-library/blob/b3be3c9ceaf9d02e8b09dc0c37b21fce590d2c27/metadata/program/src/lib.rs#L1

Reply:
oh god i didnt understand